app/views/users/show.html.erb

```diff
@@ -74,75 +74,10 @@
-                  </a>
-                <% end %>
-                <% if @user.stackoverflow_url.present? %>
-                  <a href="<%= @user.stackoverflow_url %>" target="_blank" rel="noopener me" class="px-1 align-middle inline-block">
-                    <%= inline_svg_tag("stackoverflow.svg", class: "crayons-icon", aria: true, title: "StackOverflow logo") %>
-                  </a>
-                <% end %>
-                <% if @user.dribbble_url.present? %>
-                  <a href="<%= @user.dribbble_url %>" target="_blank" rel="noopener me" class="px-1 align-middle inline-block">
-                    <%= inline_svg_tag("dribbble.svg", class: "crayons-icon", aria: true, title: "Dribbble logo") %>
-                  </a>
-                <% end %>
-                <% if @user.medium_url.present? %>
-                  <a href="<%= @user.medium_url %>" target="_blank" rel="noopener nofollow me" class="px-1 align-middle inline-block">
-                    <%= inline_svg_tag("medium.svg", class: "crayons-icon", aria: true, title: "Medium logo") %>
-                  </a>
-                <% end %>
-                <% if @user.gitlab_url.present? %>
-                  <a href="<%= @user.gitlab_url %>" target="_blank" rel="noopener nofollow me" class="px-1 align-middle inline-block">
-                    <%= inline_svg_tag("gitlab.svg", class: "crayons-icon", aria: true, title: "GitLab logo") %>
-                  </a>
-                <% end %>
-                <% if @user.instagram_url.present? %>
-                  <a href="<%= @user.instagram_url %>" target="_blank" rel="noopener nofollow me" class="px-1 align-middle inline-block">
-                    <%= inline_svg_tag("instagram.svg", class: "crayons-icon", aria: true, title: "Instagram logo") %>
-                  </a>
-                <% end %>
-                <% if @user.twitch_url.present? %>
-                  <a href="<%= @user.twitch_url %>" target="_blank" rel="noopener nofollow me" class="px-1 align-middle inline-block">
-                    <%= inline_svg_tag("twitch.svg", class: "crayons-icon", aria: true, title: "Twitch logo") %>
-                  </a>
-                <% end %>
-                <% if @user.website_url.present? %>
-                  <a href="<%= @user.website_url %>" target="_blank" rel="noopener nofollow me" class="px-1 align-middle inline-block">
-                    <%= inline_svg_tag("external.svg", class: "crayons-icon", aria: true, title: "External link icon") %>
+              <% Profile.special_social_link_attributes.each do |attribute| %>
+                <% if @user.send(attribute).present? %>
+                  <a href="<%= @user.send(attribute) %>" target="_blank" rel="noopener me" class="px-1 align-middle inline-block">
+                    <%= inline_svg_tag("#{attribute.gsub("_url","")}.svg", class: "crayons-icon", aria: true, title: "Twitter logo") %>
```

Comment:
(blocking) - We need to set the correct title for each svg here - this changes them all to be announced to screen reader users as 'Twitter logo' (incidentally, something like 'twitter profile' would be much better naming here)

Reply:
Thanks for the catch. Just pushed the fix.